.github/workflows/build.yml

```diff
@@ -62,10 +62,10 @@ jobs:
           --health-cmd="mariadb-admin ping" --health-interval=10s --health-timeout=5s
           --health-retries=5
       mssql2019:
-        image: mcmoe/mssqldocker:v2019.CU4.0
+        image: ghcr.io/uptrace/mssql:latest
```

Comment:
@vmihailenco 
I am using ghcr.io/uptrace/mssql to store this image. If this address has any other purpose, please @ me.